pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -62,17 +62,35 @@ const LOCAL_INSTALL_MEDIA_SOURCE = 'file';
 const EXISTING_DISK_IMAGE_SOURCE = 'disk_image';
 const PXE_SOURCE = 'pxe';
 
-/**
- * Returns an array of storage pools, filtered by connection and presence of volumes
+const permission = cockpit.permission({ admin: true });
+
+/* Returns available space of default storage pool
+ *
+ * First it tries to find storage pool called "default"
+ * If there is none, a pool with path "/var/lib/libvirt/images" (system connection)
+ * or "~/.local/share/libvirt/images" (session connection)
+ * If no default pool could be found, virt-install will create a pool named "default",
+ * whose available space we cannot predict
+ * see: virtinstall/storage.py - StoragePool.build_default_pool()
  *
  * @param {array} storagePools
  * @param {string} connectionName
- * @returns {array}
+ * @returns {number}
  */
-function filterStoragePools(storagePools, connectionName) {
-    return storagePools.filter(pool => pool.volumes &&
-                                       pool.volumes.length &&
-                                       pool.connectionName === connectionName);
+function getSpaceAvailable(storagePools, connectionName) {
+    let storagePool = storagePools.find(pool => pool.name === "default");
```

Comment:
This function is called only with parameter storagePools array which is already filtered by connection.
But I can also put filtering into the function itself. It would double the code at some places but function would be more generic. WDYT?

Reply:
storagePools parameter passed into that function is already filtered by connection.

Altought I could filter it in the function to make the funciton more generic.